src/js/lang.js

```diff
@@ -0,0 +1,11 @@
+var store =  {
```

Comment:
This seems overengineered :) Personally I would have added `lang` to the root data object and that's all. You can keep the function if you like (but rename it, it is not `setState` is is `setLang`)